be/src/fs/fs.cpp

```diff
@@ -68,7 +68,7 @@ inline bool is_posix_uri(std::string_view uri) {
 }
 
 inline bool is_staros_uri(std::string_view uri) {
-    return starts_with(uri, "staros_");
+    return starts_with(uri, "staros://");
```

Comment:
suggest define the prefix as a constant in fs_starlet.h, and refer to the constant variable here, to avoid future inconsistency.

Reply:
done